### PR TITLE
create column component

### PR DESCRIPTION
**Pull request by marcell-pal-ds**

### CODE REVIEW

File: core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+@ExtendWith(SlingContextExtension.class)
+public class ColumnsComponentTest {
+
+    private static final String PATH = "/content/columns";
+    private final SlingContext context = new SlingContext(ResourceResolverType.RESOURCERESOLVER_MOCK);
+
+    @BeforeEach
+    public void init() {
+        context.addModelsForClasses(CardComponent.class);
```
Comment (paweljozwicki) [inline core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java]: Probably it should be ColumnComponent.class?

File: ui.apps/src/main/jcr_root/apps/bulma/components/columns/columns.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<sly data-sly-use.model="pl.ds.bulma.components.models.ColumnsComponent">
+    <sly data-sly-test="${resource.hasChildren}">
+        <div data-sly-unwrap="${model.hasOnlyColumnsWhichHasNoChildren}" class="columns">
```
Comment (paweljozwicki) [inline ui.apps/src/main/jcr_root/apps/bulma/components/columns/columns.html]: Why do we need that unwrap? I couldn't find out it.

File: core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        context.load().json(requireNonNull(Thread.currentThread().getContextClassLoader().getResourceAsStream("columns.json")), PATH);
+    }
+
+    @Test
+    void defaultColumnsComponentModelTest() {
+        ColumnsComponent model = context.resourceResolver().getResource(PATH + "/default").adaptTo(ColumnsComponent.class);
+
+        //TODO
+        assertTrue(true);
+//        assertThat(model).isNotNull();
```
Comment (paweljozwicki) [inline core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java]: It should starts working after changing model class above.

File: core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java (hunk 1 above)
Comment (marcell-pal-ds) [reply core/src/test/java/pl/ds/bulma/components/models/ColumnsComponentTest.java]: Oh my god... I can't believe that I didn't realize that...

File: ui.apps/src/main/jcr_root/apps/bulma/components/columns/columns.html (hunk 2 above)
Comment (marcell-pal-ds) [reply ui.apps/src/main/jcr_root/apps/bulma/components/columns/columns.html]: This way there won't be an empty `div` element with the 'columns' class in the DOM of the preview page, if the columns are empty in the editor page.

Comment (paweljozwicki) [reply ui.apps/src/main/jcr_root/apps/bulma/components/columns/columns.html]: Thanks. Now I get it :)

**Review by nowak-pawel-ds — APPROVED** [review]

**Review by arturwujczak — APPROVED** [review]

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**